cmake/celix_project/AddOpenSSL.cmake

```diff
@@ -0,0 +1,39 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# include(ExternalProject)
+# ExternalProject_Add(
+#    openssl_project
+# )
+
+if (NOT TARGET OpenSSL::SSL)
+    add_library(OpenSSL::SSL GLOBAL)
+    set_target_properties(OpenSSL::SSL PROPERTIES
+        IMPORTED_LOCATION "/usr/lib/openssl/libssl.so"
+        INTERFACE_INCLUDE_DIRECTORIES "/usr/include/openssl"
+    )
+  #  set(OPENSSL_ROOT_DIR "/usr/lib/openssl")
+endif()
+
+if (NOT TARGET OpenSSL::Crypto)
+    add_library(OpenSSL::Crypto GLOBAL)
+    set_target_properties(OpenSSL::Crypto PROPERTIES
+        IMPORTED_LOCATION "/usr/lib/openssl/libcrypto.so"
```

Comment:
Does this work when OpenSSL is installed at another location, e.g. `$HOME/.local/lib/openssl/libcrypto.so`?

Reply:
This script is not loaded, so it doesn't do anything (anymore). I tried this but it does not work, I will remove the file